recipes/libx11/all/COPYING.txt

```diff
@@ -0,0 +1,942 @@
+The following is the 'standard copyright' agreed upon by most contributors,
```

Comment:
Why do you need this here? Is it not possible download from the recipe, like tools.download("link_the_license_file") ?